uploader.json

```diff
@@ -816,6 +816,11 @@
             "pattern": [],
             "symbolic_name": "ovs_vsctl_show"
         },
+        {
+            "file": "/proc/partitions",
+            "pattern": [],
+            "symbolic_name": "partitions"
+        },
```

Comment:
@akshay196 this change needs to be in the `files` section of the JSON instead of the `commands` section of the JSON.